lib/nfc/protocols/mf_classic/mf_classic_poller.c

```diff
@@ -505,6 +557,128 @@ NfcCommand mf_classic_poller_handler_request_read_sector_blocks(MfClassicPoller*
     return command;
 }
 
+NfcCommand mf_classic_poller_handler_analyze_backdoor(MfClassicPoller* instance) {
+    NfcCommand command = NfcCommandReset;
```

Comment:
Looks like this logic can be simplified and moved to mf_classic api or even to a separate module, smth like mf_classic_backdoor.c

In that module all logic of testing backdoor can be done, and as a public function there can be something like: `mf_classic_test_backdoor(MfClassicBackdoor backdoor_type);`